Gems/AtomLyIntegration/CommonFeatures/Assets/PostProcess/default.postfxlayercategories

```diff
@@ -1,17 +1,30 @@
 <ObjectStream version="3">
 	<Class name="EditorPostFxLayerCategoriesAsset" type="{A18B1B11-4C1E-4C1B-9643-178E8ED27019}">
 		<Class name="AZStd::map" field="Layer Categories" type="{5B4970CC-26DE-51A3-83E7-1BC3A7E8E3C3}">
+
```

Comment:
I definitely see the value of the blending, and the need to create a mechanism that triggers the blending (the blend itself as you mentioned is already there via the component)..
Having said that, it seems that this approach is somewhat too confined.
Why do we need any categories? why not blend any given layers using criterias such as triggers, scripts, code based logic rendering (production stitched), etc..
Having categories to do a blend seems somewhat strict and might end up not being sufficient for most productions.
Thoughts?

Reply:
Great explanation, and judging by it it might be good then to rename or add some notion of sort priority to the naming.